examples/Accelerators/ExpSinAccelerated/exp_sinusoid_accelerated.jl

```diff
@@ -0,0 +1,198 @@
+# phase ``\phi``. Given an initial guess of the parameters using some multivariate
+# normal distribution, our goal is to estimate the parameters from a noisy observation 
+# of the maximum, minimum, and mean of the true model output. We will compare the
+# parameter estimates achieved through traditional EKI and through accelerated versions
+# of the EKI algorithm.
+
+# First, we load the packages we need:
+using LinearAlgebra, Random
+using Distributions, Plots
+using EnsembleKalmanProcesses
+using EnsembleKalmanProcesses.ParameterDistributions
+const EKP = EnsembleKalmanProcesses
+fig_save_directory = @__DIR__
+
+## Setting up the model and data for our inverse problem
+dt = 0.01
+trange = 0:dt:(2 * pi + dt)
+function model(amplitude, vert_shift)
+    phi = 2 * pi * rand(rng)
+    return exp.(amplitude * sin.(trange .+ phi) .+ vert_shift)
+end
+nothing # hide
+
+# Seed for pseudo-random number generator.
+rng_seed = 41
+rng = Random.MersenneTwister(rng_seed)
+nothing # hide
+
+function G(u)
+    theta, vert_shift = u
+    sincurve = model(theta, vert_shift)
+    return [maximum(sincurve) - minimum(sincurve), mean(sincurve)]
+end
+
+function main()
+    USE_SCHEDULER = false
+    scheduler_def = DataMisfitController()
+
+    dim_output = 2
+    Γ = 0.01 * I
+    noise_dist = MvNormal(zeros(dim_output), Γ)
+    theta_true = [1.0, 0.8]
+    y = G(theta_true) .+ rand(noise_dist)
+
+    # We define a variety of prior distributions so we can study
+    # the effectiveness of momentum on this problem.
+
+    prior_u1 = constrained_gaussian("amplitude", 2, 0.1, 0, 10)
+    prior_u2 = constrained_gaussian("vert_shift", 0, 0.5, -10, 10)
+    prior = combine_distributions([prior_u1, prior_u2])
+
+    # To compare the two EKI methods, we will average over several trials, 
+    # allowing the methods to run with different initial ensembles and noise samples.
+    N_ens = 10
+    N_iterations = 15
+    N_trials = 50
+
+    # Define cost function to compare convergences. We use a logarithmic cost function 
+    # to best interpret exponential model. Note we do not explicitly penalize distance from the prior here.
+    function cost(theta)
+        return log.(norm(inv(Γ) .^ 0.5 * (G(theta) .- y)) .^ 2)
+    end
+
+    ## Solving the inverse problem
+
+    # Preallocate so we can track and compare convergences of the methods
+    all_convs = zeros(N_trials, N_iterations)
+    all_convs_acc = zeros(N_trials, N_iterations)
+    all_convs_acc_cs = zeros(N_trials, N_iterations)
+
+    for trial in 1:N_trials
+        # We now generate the initial ensemble and set up two EKI objects, one using an accelerator, 
+        # to compare convergence.
+        initial_ensemble = EKP.construct_initial_ensemble(rng, prior, N_ens)
+
+        if USE_SCHEDULER
+            scheduler = deepcopy(scheduler_def)
+            scheduler_acc = deepcopy(scheduler_def)
+            scheduler_acc_cs = deepcopy(scheduler_def)
+        else
+            scheduler = DefaultScheduler(0.1)
```

Comment:
Careful...
This loop breaks if `USE_SCHEDULER = false` 
- maybe remove the loop around `L84 - 86`
- Instead add a line at the top like defined 
```julia
scheduler_def = USE_SCHEDULER ? DataMisfitController(terminate_at=1e4) : DefaultScheduler()
```
And it is only in the Darcy example where we need the values `0.1` for the timestep.

Reply:
Yes, thank you for finding! For other (not Darcy) examples, do we want to keep default timestep of 1?

Reply:
yeah thats fine